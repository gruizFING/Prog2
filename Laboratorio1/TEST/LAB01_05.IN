hola mundo
este es el primer lab de prog2
salir