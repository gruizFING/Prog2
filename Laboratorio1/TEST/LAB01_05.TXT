hola mundo
Comando leido: hola
Con los siguientes parametros: mundo
este es el primer lab de prog2
Comando leido: este
Con los siguientes parametros: es,el,primer,lab,de,prog2
salir
Fin!