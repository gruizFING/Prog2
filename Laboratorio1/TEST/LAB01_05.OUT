Comando leido: hola
Con los siguientes parametros: mundo
Comando leido: este
Con los siguientes parametros: es,el,primer,lab,de,prog2
Fin!